content/blog/0012-effect-3.0.mdx

```diff
@@ -0,0 +1,328 @@
+---
+title: Effect 3.0 (Release)
```

Comment:
I'm missing a reference / nod to ZIO in the form of "Effect is based on proven principles, blablabla". As in "many of the fundamentals are scientifically sourced and built on giants like ZIO, etc." or something along these lines giving Effect a broader empiric evidence.

Reply:
Note it will also gives effect an air of elitism, so discounting the statement beforehand might be beneficial
"Effect doesn't try to reinvent the wheel, many before us built amazing software based on well understood and described mathematical and computer science theories. We are proud and are very grateful to be standing on the shoulders of giants such as ZIO, etc."
or something like that haha

Reply:
That would be untrue, there has been significant divergence from ZIO in terms of requests for example, and there is not a single math principle that inspire Effect, so no :)

Reply:
Then extrapolate it to match the truth my man 😭